db/manager.go

```diff
@@ -96,20 +89,25 @@ func NewManager(network net.Net, opts ...Option) (*Manager, error) {
 	return m, nil
 }
 
+// GetToken provides access to thread network tokens.
+func (m *Manager) GetToken(ctx context.Context, identity thread.Identity) (thread.Token, error) {
```

Comment:
Manager provides it's own avenue to getting a token, which just calls the method on the network layer. Note: the normal `db.go` APIs can also take a token. In these cases, the user would own the network layer, and they can call `GetToken` there directly.